### ISSUE TITLE
Inconsistent Eslint and Prettier rules

### ISSUE THREAD

**Issue by akib22** [InternetHealthReport/ihr-website#46]
Eslint and Prettier rules were added for the project but it is not consistent. Everyone has their own preferences(single/double quote, 2/4 space). If we don't force them to follow the same rule then the code style will be inconsistent. Also, `.editorconfig` rules and code styles are not the same. `.editorconfig` indent_size is 4 but in codes follows 2.

On my computer, I am getting lots of warnings for every file. Like this
![eslint waring screenshot](https://i.postimg.cc/pL0hZ8pd/Screen-Shot-2022-03-17-at-11-06-48-AM.png)

@romain-fontugne Please assign it to me, I am working on it.

**Comment by romain-fontugne** [comment]
Yes, that's a mess, sorry!
Thanks for working on that.

**Comment by Aniket762** [comment]
Thanks for pointing out @akib22, sorry from my end as I didn't check on the consistency while adding the file. 

PS. Tomar kichu lagle janiyo :)

**Comment by akib22** [comment]
Welcome both of you @romain-fontugne and @Aniket762.

Ami obbossoy janabo tomake :) @Aniket762 do you know Bengali?

**Comment by Aniket762** [comment]
> Welcome both of you @romain-fontugne and @Aniket762.
> 
> Ami obbossoy janabo tomake :) @Aniket762 do you know Bengali?

@akib22 Yups, ami bangali, abhi durgapur, west bengal, India te thaki 🙈

**Comment by akib22** [comment]
Great to hear that you are Bangali. 

> > Welcome both of you @romain-fontugne and @Aniket762.
> > Ami obbossoy janabo tomake :) @Aniket762 do you know Bengali?
> 
> @akib22 Yups, ami bangali, abhi durgapur, west bengal, India te thaki 🙈

Great to hear that you are Bangali.

**Comment by akib22** [comment]
In eslint, **"Component name "Networks" should always be multi-word"** giving this error. For details about this rule [vue/multi-word-component-names](https://eslint.vuejs.org/rules/multi-word-component-names.html).

![screenshot](https://i.ibb.co/6ZjJ5w0/Screen-Shot-2022-03-17-at-1-54-29-PM.png)

@romain-fontugne any suggestion about this error, what should I follow?

**Comment by romain-fontugne** [comment]
Yes, I have seen this for several of our components. Maybe now is a good time to rename all components that have this error.
I don't have a strong opinion about how to fix this and I am not sure about what are the best practices. 

One possibility is to rename things like this:
Api: ApiView
Contact: ContactView
Corona: CoronaReport
Countries: CountryReport
Documentation: DocumentationView
Footer: AppFooter
Header: AppHeader
Home: HomeView
Networks: NetworkReport
ROV: ROVReport

Basically everything that ends with 'View' are simple pages and those ending with 'Report' are based on dynamically generated content from the API.
But as I said I am open to suggestions.

**Comment by akib22** [comment]
Thank you, I am going with your suggestions.

Inside `src/components/ripe/` directory, `initTracemon`, `ripestat`, and `initLatencymon` functions are uesed. I can't find any defination for those function. I think those are errors. Can you please clarify @romain-fontugne? Also please suggest name for those components. 

![code screenshot](https://i.ibb.co/p2SWkgn/Screen-Shot-2022-03-19-at-1-19-43-AM.png)

**Comment by romain-fontugne** [comment]
All these functions are defined in libraries that are loaded when needed. For example, initLatencymon is defined here  "https://atlas.ripe.net/resource/latencymon/latencymon-widget-main.js". See src/main.js. 
I guess we can just ignore these errors.

Let's rename them like this:
BgplayWidget
LatencymonWidget
TracemonWidget

Thanks!

**Comment by akib22** [comment]
Thank you for your clarifications. I ignored those errors and updated those component names. As well as open the #54 for review.

**Comment by romain-fontugne** [comment]
Thanks, the code looks a lot better. I merged the pull request, there was two 'test' text in GlobalReport I have removed them otherwise it looked good. 

But I don't see the components renaming? have pushed your latest changes?

**Comment by akib22** [comment]
Sorry for the late response. 
Actually, I add the component name inside the script tag instead of renaming the filename. Because If I rename the filenames, then there will have a chance to remove others' latest changes.

**Comment by romain-fontugne** [comment]
OK, I see. thanks!
